EIPS/eip-5202.md

```diff
@@ -48,7 +52,8 @@ A blueprint contract MAY insert any bytes (data or code) between the version byt
 
 
 ## Backwards Compatibility
-Needs discussion
+
+No known issues
 
 ## Reference Implementation
```

Comment:
Is there a reason that the Reference implementation only includes how to verify a blueprint's preamble, and not how to construct one? I think this would be very helpful, rather than having to dig through someone like Lido's scripts, a short example could be provided

Reply:
added in a524dca3f5340fd152c93c10be17d0e14d640ba7